addons/website_forum/tests/test_performance.py

```diff
@@ -34,7 +34,7 @@ def test_perf_sql_forum_standard_data(self):
         number_of_queries = self._get_url_hot_query(self.forum._compute_website_url(), cache=False)
         self.assertLessEqual(number_of_queries, 28)
         number_of_queries = self._get_url_hot_query(self.post.website_url)
-        self.assertEqual(number_of_queries, 21)
+        self.assertEqual(number_of_queries, 20)
```

Comment:
Now, you added again `'seo_microdata': question._get_microdata(),` line, I think we no longer need to change this line

Reply:
Hello @xavierbol :wave:

Apologies :pray: 
I've updated the code